assets/src/vo/money.js

```diff
@@ -237,7 +223,7 @@ export default class Money {
 	 * @return {Money} Returns a new instance of Money.
 	 */
 	add( other ) {
-		Money.assertEquivalentWithCurrency( this, other );
+		Money.assertUsingEqualCurrency( this, other );
```

Comment:
better... but still think `assertUsingSameCurrency()` is even **more** clear that there are zero differences